src/test/java/com/zendesk/maxwell/BootstrapIntegrationTest.java

```diff
@@ -0,0 +1,181 @@
+package com.zendesk.maxwell;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class BootstrapIntegrationTest extends AbstractMaxwellTest {
+
+	@Test
+		public void testSingleRowBootstrap( ) throws Exception {
+		List<RowMap> list;
+		String input[] = {
+			"insert into minimal set account_id = 1, text_field='hello'",
+			"insert into maxwell.bootstrap set database_name = 'shard_1', table_name = 'minimal'"
```

Comment:
Is it worth adding a test for bootstraps of multiple tables? Although I suppose it might get hard if there are ordering issues (multiple bootstraps going on simultaneously).